LayoutTests/fast/css3-text/css3-text-decoration/text-decoration-dashed.html

```diff
@@ -3,6 +3,6 @@
 <head>
 </head>
 <body>
-<div style="font-size: 40px; -webkit-text-decoration-style: dashed; text-decoration: underline;">dotyedgdesc</div>
+<div style="font-size: 40px; text-decoration: underline; text-decoration-style: dashed;">dotyedgdesc</div>
```

Comment:
I think this is good but I wonder if we should just use the longhands here as we are not worried about testing the shorthand.